.github/workflows/go-presubmit.yml

```diff
@@ -117,7 +117,7 @@ jobs:
       - name: setup kind
         uses: engineerd/setup-kind@v0.5.0
```

Comment:
do we need to upgrade this version?

Reply:
@qiujian16  v0.5.0 is latest https://github.com/engineerd/setup-kind/tags

Reply:
e2e fails..